footprints/main/tests/test_views.py

```diff
@@ -1169,6 +1169,30 @@ def test_empty_search(self):
         self.assertFalse(response.context['search_criteria'])
 
 
+class ToggleGalleryViewTest(TestCase):
+
+    def test_list_view_search(self):
+        url = reverse('search')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertFalse(response.context['gallery_view'])
+
+        url = "{}/?q=a".format(reverse('search'))
```

Comment:
Not sure if this check from Ln. 1180 is really needed?

Reply:
I was getting instances where it would default to Gallery View when using the pagination buttons. I can delete it if you like.